evennia/contrib/aws-s3-cdn.py

```diff
@@ -0,0 +1,847 @@
+"""
+This plugin migrates the Web-based portion of Evennia,
+namely images, javascript, and other items located
+inside staticfiles into Amazon AWS (S3) for hosting.
+
+INSTALLATION:
+
+1) If you don't have an AWS S3 account, you should create one now.
```

Comment:
I think it would be a good idea with a link or even a short description for how you can do this, and what it will cost (or not).